docs/api/lib-export.adoc

```diff
@@ -0,0 +1,173 @@
+= Export Library
+
+:toc: right
+:imagesdir: ../images
+
+image:xp-780.svg[XP 7.8.0,opts=inline]
+
+This library provides functions for generation node exports and importing them.
+
+== Usage
+
+Add the following into your `build.gradle` file:
+
+[source,groovy]
+----
+dependencies {
+  include "com.enonic.xp:lib-export:${xpVersion}"
+}
+----
+
+In your JavaScript controller, add a require statement:
+
+[source,js]
+----
+const exportLib = require('/lib/xp/export');
+----
+
+You are now ready to use the library functionality.
+
+== Functions
+
+=== exportNodes
+
+Export nodes to a nodes-export.
+Export is created in <<../deployment/config#_export, exports>> directory.
+
+[.lead]
+Parameters
+
+An object with the following keys and their values:
+
+[%header,cols="1%,1%,1%,1%,98%a"]
+[frame="none"]
+[grid="none"]
+|===
+| Name | Type | Attributes | Default | Description
+
+| sourceNodePath | string | | | Source nodes path
+| exportName | string | | | Export name
```

Comment:
Missing `targetDirectory`

Reply:
there is no target directory in js API.. Deliberately.